R/provider-openai.R

```diff
@@ -200,8 +200,9 @@ method(chat_params, ProviderOpenAI) <- function(provider, params) {
       frequency_penalty = "frequency_penalty",
       presence_penalty = "presence_penalty",
       seed = "seed",
-      max_tokens = "max_completion_tokens",
+      max_completion_tokens = "max_tokens",
       logprobs = "log_probs",
+      top_logprobs = "top_k",
```

Comment:
This threw us off a bit! These mappings were backwards (and the `top_k` parameter was missing) in the source, which caused us to initially write the mappings backwards.

I fixed this here and left a template in `params.R` for future reference: 1ecb72929703932a56bf21ea3fa9fe889c662d3e

Reply:
Ooops